pkg/minikube/node/cache.go

```diff
@@ -99,13 +99,17 @@ func doCacheBinaries(k8sVersion string) error {
 }
 
 // BeginDownloadKicArtifacts downloads the kic image + preload tarball, returns true if preload is available
-func beginDownloadKicArtifacts(g *errgroup.Group) {
```

Comment:
cRuntime is not used here

Reply:
I thought it would be, but we don't use the preload hack for the kicbase image